src/main/resources/reference.conf

```diff
@@ -156,4 +156,46 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+cassandra-query-journal {
+  # Implementation class of the Cassandra ReadJournalProvider
+  class = "akka.persistence.cassandra.query.journal.CassandraReadJournalProvider"
+
+  # Stream queries database in defined `refresh-interval`.
+  refresh-interval = 3s
+
+  # How many events to fetch in one query (replay) and keep buffered until they
+  # are delivered downstreams.
+  max-buffer-size = 100
+
+  # Target partition size of index tables. Must match cassandra-journal configuration.
+  target-partition-size = 500000
```

Comment:
We were considering that, but decided to keep them completely separate. The reason is that a client might want to use persistence query, but has no intention of using write journal. 

Target partition size in particular however is important to be the same in both. So we could create a shared config section, similarly to the config code representation.